tests/helpers/test_entity.py

```diff
@@ -663,10 +663,9 @@ async def test_set_context_expired(hass: HomeAssistant) -> None:
     """Test setting context."""
     context = Context()
 
-    with patch.object(
-        entity.Entity, "context_recent_time", new_callable=PropertyMock
-    ) as recent:
-        recent.return_value = timedelta(seconds=-5)
+    with patch(
+        "homeassistant.helpers.entity.CONTEXT_RECENT_TIME", timedelta(seconds=-5)
```

Comment:
Why is it negative?

Reply:
By setting it to a negative time, the context is never valid.

It's a stupid test, it would be better to:
- freeze time
- set a context
- bump time by 5 seconds
- verify the context is no longer valid